### PR TITLE
Modify certification profile and fixing the test failures on FMW pipeline jenkins

### PR DESCRIPTION
**Pull request by vanajamukkara**
Modify certification profile 
fixing the test failures to run on FMW pipeline jenkins
Adding checks for kubectl download in script

FMW Jenkins results -
Sequential run - https://ci-cloud.us.oracle.com/jenkins/wlsqa/job/wko-tmp/121/ - 1 failure, not related to changes in this PR

WKO Jenkins -
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/6321/ - 5 failures, not related to changes in this PR

### CODE REVIEW

File: kindtest.sh [hunk 1]
```diff
@@ -308,7 +308,7 @@ if [ ${test_filter} != "**/It*" ]; then
   echo "Running mvn -Dit.test=${test_filter} -Dwdt.download.url=${wdt_download_url} -Dwit.download.url=${wit_download_url} -Dwle.download.url=${wle_download_url} -DPARALLEL_CLASSES=${parallel_run} -DNUMBER_OF_THREADS=${threads}  -pl integration-tests -P ${maven_profile_name} verify"
   time mvn -Dit.test="${test_filter}" -Dwdt.download.url="${wdt_download_url}" -Dwit.download.url="${wit_download_url}" -Dwle.download.url="${wle_download_url}" -DPARALLEL_CLASSES="${parallel_run}" -DNUMBER_OF_THREADS="${threads}" -pl integration-tests -P ${maven_profile_name} verify 2>&1 | tee "${RESULT_ROOT}/kindtest.log" || captureLogs
 else
-  if [ "${maven_profile_name}" = "wls-image-cert" ] || [ "${maven_profile_name}" = "fmw-image-cert" ] || [ "${maven_profile_name}" = "kind-sequential" ]; then
+  if [ "${maven_profile_name}" = "wls-image-cert" ] || [ "${maven_profile_name}" = "new-wls-image-cert" ] || [ "${maven_profile_name}" = "fmw-image-cert" ] || [ "${maven_profile_name}" = "kind-sequential" ]; then
```
Comment (sankarpn) [inline kindtest.sh:311]: Is the wls-cert-image still there? why not have just one cert image profile?

**Review by anpanigr — APPROVED** [review]
lgtm

File: kindtest.sh (hunk 1 above)
Comment (vanajamukkara) [reply kindtest.sh]: modified to have one profile

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**